### DISCUSSION TITLE
Hope to share the Demo program(preview.gif)

### DISCUSSION THREAD

**Discussion by Laboputer** [Live-Charts/LiveCharts2#1183] [Q&A] [answered]
Would you like to share the demo program in the preview gif image introduced in Readme? 
I'm so curious about what controls were used to design each theme.
I think it will help users how to use it.
Thanks & Regards.

**Comment by beto-rodriguez** [comment, marked as answer]
Hello, it will be added soon, RC1 will be released soon, and all the samples will be added to the site.

**Reply by Laboputer** [reply]
I'll be waiting for it to be added. Thank you for your reply!
